### PR TITLE
NX: scripts configs for cloud caches

### PR DESCRIPTION
**Pull request by iBat**

### CODE REVIEW

File: apps/angular/project.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "devextreme-angular-playground",
```
Comment (pomahtri) [inline apps/angular/project.json:2]: Let's delete `project.json` file for all playgrounds
I think default inputs and outputs will look more simple and more stable

File: apps/demos/project.json [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  "projectType": "application",
+  "targets": {
+    "test": {
+      "executor": "nx:run-script",
+      "options": {
+        "script": "test"
+      },
+      "dependsOn": [
+        "!^build"
+      ],
```
Comment (pomahtri) [inline apps/demos/project.json]: Does nx support excluding targets from `dependsOn`? Shouldn't it be like so?
```suggestion
      "dependsOn": [],
```

File: e2e/testcafe-devextreme/project.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "devextreme-testcafe-tests",
```
Comment (pomahtri) [inline e2e/testcafe-devextreme/project.json:2]: Same as for playgrouds, let's delete this file. Looks like defaults target settings are OK

File: apps/angular/project.json (hunk 1 above)
Comment (iBat) [reply apps/angular/project.json:2]: Inputs - may be. But outputs - no.

File: e2e/testcafe-devextreme/project.json (hunk 3 above)
Comment (pomahtri) [reply e2e/testcafe-devextreme/project.json:2]: resolved in DM, decided to keep settings explicitly

**Review by pomahtri — APPROVED** [review]

**Pull request merged**